frontend/packages/shared/src/pure/utils.ts

```diff
@@ -1,5 +1,5 @@
 export const isNumeric = (str: string) => parseInt(str).toString() === str;
-export const deepCopy = (value: any) => JSON.parse(JSON.stringify(value));
+export const deepCopy = <T>(value: T) => JSON.parse(JSON.stringify(value)) as T;
```

Comment:
Kan vurdere om vi skal bytte ut JSON.parse(JSON.stringify()); med structuredClone? Hva tenker du om det? :)

```suggestion
export const deepCopy = <T>(value: T) => structuredClone(value) as T;
```

Reply:
Den kjente jeg ikke til, godt forslag!

Reply:
https://github.com/jsdom/jsdom/issues/3363#issuecomment-1467894943, tenker vi kan utsette å gjøre noe med dette for nå. 👍